llvm/test/Instrumentation/MemorySanitizer/i386/msan_i386_bts_asm.ll

```diff
@@ -4,7 +4,7 @@
 ; RUN: opt < %s -msan-kernel=1 -msan-check-access-address=0 -msan-handle-asm-conservative=1 -S -passes=msan 2>&1 | FileCheck %s -check-prefixes=CONS
 
 target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
```

Comment:
I was fixing up few minor issue, and noticed that 
`target datalayout =` is still 64 bit, generating wrong IR

Could you please fix, and I'll merge tomorrow?

Reply:
I guess we can merge right now. Please create a new PR with updated datalayout and tests

Reply:
Thank you! I will open follow up PR's later today.

Reply:
Sorry for the delay. Didn't have time to open PR's yet. Will do this week.